DrivewayDentDeletion/Operators/cicd-test-apic/cicd-pipeline.yaml

```diff
@@ -0,0 +1,471 @@
+          workspace: git-source
+
+    # deploy all ace bernie in test namespace
+    - name: deploy-wait-ace-bernie-test
+      runAfter:
+        - image-push-to-test-ace-bernie
+      taskRef:
+        # name: deploy-test-task
+        name: deploy-task
+      params:
+        - name: imageTag
+          value: "$(params.imageTag)-test"
+        - name: imageName
+          value: "ace-bernie"
+        - name: releaseName
+          value: "ace-bernie-int-srv"
+        - name: releaseScript
+          value: release-ace-integration-server.sh
+        - name: namespace
+          value: "{{NAMESPACE}}-ddd-test"
+      workspaces:
+        - name: git-source
+          workspace: git-source
+
+    # deploy all ace acme in test namespace
+    - name: deploy-wait-ace-acme-test
+      runAfter:
+        - image-push-to-test-ace-acme
+      taskRef:
+        # name: deploy-test-task
+        name: deploy-task
+      params:
+        - name: imageTag
+          value: "$(params.imageTag)-test"
+        - name: imageName
+          value: "ace-acme"
+        - name: releaseName
+          value: "ace-acme-int-srv"
+        - name: releaseScript
+          value: release-ace-integration-server.sh
+        - name: namespace
+          value: "{{NAMESPACE}}-ddd-test"
+      workspaces:
+        - name: git-source
+          workspace: git-source
+
+    # deploy all ace chris in test namespace
+    - name: deploy-wait-ace-chris-test
+      runAfter:
+        - image-push-to-test-ace-chris
+      taskRef:
+        # name: deploy-test-task
+        name: deploy-task
+      params:
+        - name: imageTag
+          value: "$(params.imageTag)-test"
+        - name: imageName
+          value: "ace-chris"
+        - name: releaseName
+          value: "ace-chris-int-srv"
+        - name: releaseScript
+          value: release-ace-integration-server.sh
+        - name: namespace
+          value: "{{NAMESPACE}}-ddd-test"
+      workspaces:
+        - name: git-source
+          workspace: git-source
+    - name: apic-resource-config-dev
+      runAfter:
+        - clone-git-source
+      taskRef:
+        name: apic-resource-config
+      params:
+        - name: environment
+          value: "dev"
+        - name: releaseName
+          value: "ademo"
+        - name: releaseScript
+          value: publish-api.sh
+      workspaces:
+        - name: git-source
+          workspace: git-source
+    - name: apic-resource-config-test
+      runAfter:
+        - test-e2e-api-in-dev-namespace
+      taskRef:
+        name: apic-resource-config
+      params:
+        - name: environment
+          value: "test"
+        - name: releaseName
+          value: "ademo"
+        - name: releaseScript
+          value: publish-api.sh
+      workspaces:
+        - name: git-source
+          workspace: git-source
+
+    # test end-to-end API in test namespace
+    - name: test-e2e-api-in-test-namespace
+      runAfter:
+        - deploy-wait-ace-chris-test
```

Comment:
I think this should run after `apic-resource-config-test`

Reply:
yes. Once we have the script for it we will change it. Or much better we plan to update the current test-api script to have APIC option also. So in the next PR we will change it. this was for now to have e2e run